The cmath header must be included in ELC.h order to correctly select the
overloaded abs() function.

Author: Davide Vanzo (Vanderbilt University)
diff -ru MSM_HOCR-1.0.orig/extras/ELC1.04/ELC/ELC.h MSM_HOCR-1.0/extras/ELC1.04/ELC/ELC.h
--- MSM_HOCR-1.0.orig/extras/ELC1.04/ELC/ELC.h	2019-07-03 17:26:05.296883529 -0500
+++ MSM_HOCR-1.0/extras/ELC1.04/ELC/ELC.h	2019-07-08 11:47:28.545974054 -0500
@@ -156,6 +156,7 @@
 #include <iterator>
 #include<assert.h>
 #include <limits>
+#include <cmath>
 
 namespace ELCReduce {
 
